tests/acceptance/test_organization_security_privacy.py

```diff
@@ -56,3 +58,52 @@ def test_setting_2fa_without_2fa_enabled(self):
         self.browser.wait_until_not(".modal")
         self.browser.wait_until_test_id("toast-error")
         self.load_organization_helper("setting 2fa without 2fa enabled")
+
+    def test_renders_advanced_data_scrubbing_without_rule(self):
+        user_owner = self.create_user("owner@example.com")
+        organization = self.create_organization(name="Example", owner=user_owner)
+        self.login_as(user_owner)
+        path = "/settings/{}/security-and-privacy/".format(organization.slug)
+
+        self.browser.get(path)
+        self.browser.wait_until_not(".loading-indicator")
+        assert self.browser.element_exists('[data-test-id="advanced-data-scrubbing"]')
+
+        self.load_organization_helper("advanced-data-scrubbing-without-rule")
+
+    def test_renders_advanced_data_scrubbing_with_rules(self):
+        user_owner = self.create_user("owner@example.com")
+        organization = self.create_organization(name="Example", owner=user_owner)
+        relayPiiConfig = json.dumps(
+            {
+                "rules": {
+                    "0": {
+                        "type": "password",
+                        "redaction": {"method": "replace", "text": "Scrubbed"},
+                    },
+                    "1": {"type": "creditcard", "redaction": {"method": "mask"}},
+                },
+                "applications": {"password": ["0"], "$message": ["1"]},
+            }
+        )
+        organization.update_option("sentry:relay_pii_config", relayPiiConfig)
+        self.login_as(user_owner)
+        path = "/settings/{}/security-and-privacy/".format(organization.slug)
+
+        self.browser.get(path)
+        self.browser.wait_until_not(".loading-indicator")
+        assert self.browser.element_exists('[data-test-id="advanced-data-scrubbing"]')
+        assert self.browser.element_exists('[data-test-id="advanced-data-scrubbing-rules"]')
```

Comment:
Generally, instead of asserting that an element exists, I would instead use `self.browser.wait_until` as this would account for flakeyness if it takes a bit longer to render.